aioshelly/wsrpc.py

```diff
@@ -108,7 +110,10 @@ async def disconnect(self):
 
         self._rx_task = None
 
-    async def _handle_call(self, frame_id):
+    async def _handle_call(self, frame_id: str) -> None:
+        if not self._client:
+            return
```

Comment:
This can't happen so it is better to assert here and not just return

```suggestion
        assert self._client
```